scripts/aws

```diff
@@ -8,11 +8,14 @@ set -o pipefail
 trap - INT TERM
 
 docker run --rm \
-  -t "$(tty &>/dev/null && echo "-i")" \
-  -e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
-  -e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
-  -e "AWS_REGION=${AWS_REGION}" \
-  -e "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" \
-  -v "$(pwd):/project" \
-  mesosphere/aws-cli \
-  "$@"
+	-t $(tty &>/dev/null && echo "-i") \
```

Comment:
the quotes here were suggested by shellcheck - I *think* this works fine with them

Reply:
Given the use case (i.e. non-interactive ci build) we could drop the `-t -i` all together. Even on one of our laptops we'd be using `aws-vault` so the interactive bit on the aws container is not needed.

Reply:
Also .. hrm. I know what shellcheck is doing but it's wrong : )

Putting the quotes around that will result in `-t "-i"` which would try to pass the _string_ `-i` to the `-t` arg. Which, one, won't work and two, isn't desired. I'd leave it as:
```
	-t $(tty &>/dev/null && echo "-i") \
```
This will result in the appropriate `-t -i`.

Reply:
that makes a lot of sense yep,  shell param parsing is so icky :)